packages/shared/src/hooks/useDeleteSquad.ts

```diff
@@ -12,7 +12,7 @@ interface UseDeleteSquadModal {
 }
 
 type UseDeleteSquadProps = {
-  squad: Squad;
+  squad: Partial<Squad>;
```

Comment:
Bear with me, I will introduce a PR to refactor the Squad form as it is quite difficult to manage/maintain.